inputfiles/removedTypes.json

```diff
@@ -21,6 +21,16 @@
     },
     {
         "kind": "method",
+        "interface": "URL",
+        "name": "createObjectURL"
```

Comment:
these should be added on a new interface `URLConstructor`, and `var URL: URLConstructor`